src/netex-convertor/handler.ts

```diff
@@ -3,7 +3,7 @@ import singleTicketNetexGenerator from './single-ticket/singleTicketNetexGenerat
 import periodTicketNetexGenerator from './period-ticket/periodTicketNetexGenerator';
 import * as db from './data/auroradb';
 import * as s3 from './data/s3';
-import { MatchingData, PeriodTicket } from './types';
+import {MatchingData, MatchingReturnData, PeriodTicket} from './types';
 
 export const netexConvertorHandler = async (event: S3Event): Promise<void> => {
     try {
```

Comment:
(Mentioned in a comment in a different file, but we need to update the types used in here)